nengo/tests/conftest.py

```diff
@@ -157,7 +157,10 @@ def function_seed(function, mod=0):
     hash_list = os.path.normpath(path).split(os.path.sep) + [c.co_name]
     hash_string = ensure_bytes('/'.join(hash_list))
     i = int(hashlib.md5(hash_string).hexdigest()[:15], 16)
-    return (i + mod) % npext.maxint
+    s = (i + mod) % npext.maxint
+    int_s = int(s)  # numpy 1.8.0 bug when RandomState on long type inputs
+    assert int_s == s  # no overflow because of mod by npext.maxint
```

Comment:
This does not work as you would think. [The documentation of `int()` says](https://docs.python.org/2/library/functions.html#int)
>  If the argument is outside the integer range, the function returns a long object instead.
Thus, the assert will never be false.